package.json

```diff
@@ -3,15 +3,18 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
-  "type": "module",
   "scripts": {
-    "test": "jest --coverage"
+    "test": "jest --coverage",
+    "lint": "eslint test/** --max-warnings=0"
```

Comment:
Configured the lint command to only check test folder and moved the dummy test there

Reply:
Remember to also include this in .github/workflows/node.js.yml so that it runs also in the pipeline.

Reply:
Added, good catch